### DISCUSSION TITLE
Is there a native way to hide navigation when there's nothing to slide?

### DISCUSSION THREAD

**Discussion by Draghmar** [nolimits4web/swiper#5015] [Q&A] [answered]
Situation is very simple. I have few elements that don't occupy container's space when on higher resolutions. So there's nothing to slide at that moment. On lower resolutions there's not enough space for them so the sliding is needed. When using navigation I'm getting both arrows and bullet in the first scenario when they aren't needed there. I know I could track events and to this "manually" but I wonder if there's a native solution for that. Like "Hide navigation when there's only one page" or something similar.

**Comment by nolimits4web** [comment, marked as answer]
`watchOverflow` https://swiperjs.com/swiper-api#param-watchOverflow

**Reply by Draghmar** [reply]
And here I thought I searched documentation with "nav" keyword...Thanks!

**Reply by jpierson-at-riis** [reply]
Even after reading I'm not really sure why this behavior is called `watchOverflow`... If anything it seems like "underflow" would make more sense in this situation. Naming is hard, and perhaps this name makes more sense than I realize.
